From 6fdf900f79413fd17d519973bfabca6631e756ba Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Wed, 24 Oct 2012 11:58:53 -0500
Subject: [PATCH 2230/2463] ENGR00231266-4: board-mx6q_sabreauto adv7280
 csi-tx slave address

* Pass csi-tx slave address for adv7280 chipset

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index a3e9af5..649ead2 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -643,6 +643,8 @@ static struct fsl_mxc_tvin_platform_data adv7280_data = {
 	.pvdd_reg	= NULL,
 	.pwdn		= NULL,
 	.cvbs		= true,
+	/* csi slave reg address */
+	.csi_tx_addr = 0x51,
 };
 
 static struct imxi2c_platform_data mx6q_sabreauto_i2c2_data = {
-- 
1.7.7.4

